olp-cpp-sdk-authentication/src/AuthenticationCredentials.cpp

```diff
@@ -64,6 +64,9 @@ AuthenticationCredentials::ReadFromStream(std::istream& stream) {
   std::string line;
 
   while (std::getline(stream, line)) {
+    // Handle both CRLF and LF line endings
+    line.erase(line.find_last_not_of("\r\n") + 1);
```

Comment:
I think it's better not to hardcode `\r\n` but remove all std::isspace characters from the end